joint_names: ["arm_1_joint","arm_2_joint","arm_3_joint","arm_4_joint","arm_5_joint","arm_6_joint"]

# back side positions
home: [[-0.54105206811824214, -1.9198621771937625, 2.0071286397934789, 0.0, 1.4311699866353502, 0.087266462599716474]]

pickup1: [[-0.47298423394836603, -1.1030481005762216, 1.6353734558554649, 0.010471975928084897, 1.0297442586766543, 0.35604716074894049]]
pickup2: [[-0.47298423394836603, -1.1414453574359025, 1.6057029118347832, 0.010471975928084897, 1.0978121261360441, 0.35604716074894049]]
pickup3: [[-0.47298423394836603, -1.2130038301360591, 1.5184364492350666, 0.0087266462599716477, 1.2548917588155337, 0.35779249665883756]]

intermediate1: [[-0.54105206811824214, -1.9198621771937625, 2.0071286397934789, 0.0, 1.4311699866353502, 0.087266462599716474]]
intermediate2: [[0.46949356212857191, -1.9198621771937625, 2.0071286397934789, 0.0, 1.4311699866353502, 0.087266462599716474]]

drop4: [[0.51138145751853314, -1.3508848676752219, 1.4765485205555919, -0.024434609111801692, 1.4922565104551517, 2.7349308846118925]]
drop3: [[0.47123889803846897, -1.2304571226560024, 1.4957471156959186, -0.0034906585560035243, 1.3142329800149519, 2.8413960755099903]]
drop2: [[0.47123889803846897, -1.1466812652970526, 1.60919351707555, -0.0034906585560035243, 1.1152654186559874, 2.843141244972319]]
drop1: [[0.47123889803846897, -1.1065387723960158, 1.6388641942542863, -0.0034906585560035243, 1.0454522485762143, 2.843141244972319]]


# trajectories
pickup: [pickup1,pickup2,pickup3]
drop: [drop4,drop3,drop2,drop1]
